packer/provisioner_test.go

```diff
@@ -1,9 +1,14 @@
+// +build !race
+
```

Comment:
```suggestion
```
If the race detection is unhappy I think it would be better to fix the cause instead of the warning. 🙂

Reply:
Fixed this issue.